test/integration/sanity/Sanity.e2e.test.js

```diff
@@ -0,0 +1,20 @@
+// <reference types="Cypress" />
+describe('Preview Sanity', () => {
+    let token;
+
+    beforeEach(() => {
+        token = Cypress.env('ACCESS_TOKEN');
```

Comment:
Seems like this could be made a const and done just inside `describe` above unless token is modified in tests.